src/Resources/views/Admin/UiElement/text.html.twig

```diff
@@ -3,7 +3,9 @@
     type: text
     element fields :
         content
+        align
 #}
-{% if element is not empty and element.content is not empty %}
```

Comment:
Why removing this condition?

I think it's still a requirement to have it since we are not 100% sure of the content.
It's also good for compatibility with the v1.

Reply:
It's an error I restored it